server.tests/ControllerTests/LoginControllerTests.cs

```diff
@@ -27,17 +28,65 @@ public void AuthenticateUserReturnNullIfUnableReturnsDesiredUser()
             // Arrange
             User user = new User("John", "Doe", "john.doe@email.com", "supersecurepassword");
             loginService.Setup(x => x.GetUser(user.Email)).Returns(user);
-            LoginController loginController = new LoginController(loginService.Object,  config.Object);
+            LoginController loginController = new LoginController(loginService.Object, config.Object);
             // Act
             var result = loginController.AuthenticateUserReturnNullIfUnable(user.Email, user.Password);
             // Assert
             Assert.Equal(user, result);
         }
 
-        // Util methods
-        private void setupMock()
+        [Fact]
+        public void AuthenticateUserReturnNullIfUnableReturnsNull()
         {
+            // Arrange
+            User user = new User("John", "Doe", "john.doe@email.com", "supersecurepassword");
```

Comment:
Is dit nog temporary of ???

Reply:
Nope, is voor testen. Daarbij maakt het daadwerkelijke wachtwoord niet uit, want het is toch om te testen.